examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -128,6 +129,12 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback,
                 mapboxNavigation.setRoutes(emptyList())
             }
         }
+//        TODO: Remove after testing rl-237-sdk-nn-sync-in-polling
```

Comment:
Do we want to remove these code as testing has been done already or do we want to leave it as could help testing / debugging in the future? 🤔 Same question applies to the rest of `TODO`s added to `SimpleMapboxNavigationKt` 👇

Reply:
Good question, Pablo! @mskurydin What do you think? Do we need to leave red/blue dots for raw/enhanced location for testing? Or we can remove them?

Reply:
What about adding it to a new `DebugMapboxNavigationKt` example and leave `SimpleMapboxNavigationKt` as it was (which is currently far from "simple" 😅)?

Reply:
@Guardiola31337 👍 Good idea)